src/connect/marlin_printer.cpp

```diff
@@ -257,6 +258,21 @@ Printer::Params MarlinPrinter::params() const {
     params.filament_used = Odometer_s::instance().get(Odometer_s::axis_t::E);
     params.has_usb = marlin_vars->media_inserted;
 
+    struct statvfs fsbuf = {};
+    if (params.has_usb && statvfs("/usb/", &fsbuf) == 0) {
+        // Contrary to the "unix" documentation for statvfs, our FAT implementation stores:
+        // * Number of free *clusters*, not blocks in bfree.
+        // * Number of blocks per cluster in frsize.
```

Comment:
I have a strange feeling, that blocks vs. clusters don't have a well defined meaning. And it gets worse :wink: :
- "traditional" ext2,3,4 filesystems use the word "blocks" for an allocation unit
- FAT filesystems on the other hand use the word "clusters" for the same thing, but a block size is also often referenced here - usually it describes the size of an allocation unit on the physical device (a cluster contains is made of 1 or more physical blocks)

So IMO the `statvfs` documentation is on par with our implementation, it is "just" the traditional confusion between clusters and blocks on FAT filesystems.